src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/PlayerInfo.java

```diff
@@ -25,9 +25,9 @@
 				for (Info info : infos) {
 					MiscSerializer.writeUUID(serializer, info.uuid);
 					VarNumberSerializer.writeSVarInt(serializer, 0); //entity id
-					StringSerializer.writeString(serializer, version, info.getName());
+					StringSerializer.writeString(serializer, version, info.username);
 					StringSerializer.writeString(serializer, version, "Standard_Steve");
-					ByteArraySerializer.writeByteArray(serializer, version, PESkin.STEVE);
+					ByteArraySerializer.writeByteArray(serializer, version, PESkin.STEVE); //Meh.
```

Comment:
You're right, but we've only implemented Steve for now. All players are Steves :)